src/fdtransfer/fdtransferServer.cpp

```diff
@@ -68,7 +68,13 @@ bool FdTransferServer::bindServer(struct sockaddr_un *sun, socklen_t addrlen, in
         }
     }
 
-    if (bind(_server, (const struct sockaddr*)sun, addrlen) < 0 || listen(_server, 1) < 0) {
+    if (bind(_server, (const struct sockaddr*)sun, addrlen) < 0) {
```

Comment:
I split it here because I kept getting `FdTransfer listen()` errors, and some error codes overlap between `listen` and `bind`, so best to keep them as separate messages.